client/src/pages/ProfileSetting/PersonalInformationForm/PersonalInformationForm.tsx

```diff
@@ -0,0 +1,119 @@
+import { FunctionComponent } from 'react';
+import { useAuth } from '../../../context/useAuthContext';
+import { Button, TextField, Grid, CircularProgress, FormLabel, Box, Paper } from '@material-ui/core';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import updatePersonalInformation from '../../../helpers/APICalls/updatePersonalInformation';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+
+import useStyles from './useStyles';
+
+interface UserPersonalInformation {
+  headline: string;
+  bio: string;
+  location: string;
+}
+const PersonalInformationForm: FunctionComponent = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const formSubmitHandler = (
+    { headline, bio, location }: UserPersonalInformation,
+    { setSubmitting }: FormikHelpers<UserPersonalInformation>,
+  ) => {
+    if (loggedInUser) {
+      updatePersonalInformation(loggedInUser.email, headline, bio, location).then((data) => {
+        if (data.error) {
+          updateSnackBarMessage(data.error.message);
+        } else {
+          updateSnackBarMessage('Saved');
```

Comment:
Maybe consider updating the context once the controller on the BE has been modified?

Reply:
Same concern with 'useAuthContext' as above